### PR TITLE
add github-henryho

### PR DESCRIPTION
**Pull request by ecshenryho**

### CODE REVIEW

File: _pins/github-henryho [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle:ecshenryho
```
Comment (crichID) [inline _pins/github-henryho]: @ecshenryho you will need to have a space between the `:` and your username!

File: _pins/github-henryho [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Latitude: 33.825464
```
Comment (crichID) [inline _pins/github-henryho]: @ecshenryho both Latitude and Longitude should start with a lowercase l.

**Comment by krudge** [comment]
Nice!

**Comment by ChasSykes** [comment]
Hi ecshenryho,

I think another roadblock to successful build might be that the filename is supposed to be your handle and the extension ".json". Try naming your file "ecshenryho.json", in addition to whatever other recommendations were made before mine.

**Comment by brianamarie** [comment]
@ecshenryho  I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**